src/ducks/stats/services.js

```diff
@@ -0,0 +1,36 @@
+import moment from 'moment'
+import { BankTransaction } from 'cozy-doctypes'
+import { sumBy } from 'lodash'
+
+export const getPeriod = () => {
+  const end = moment()
+    .utc()
```

Comment:
I've done my example in UTC yesterday but Patrick tolds me you have all your transactions written un Europe/Paris so that's probably what you want to ask here

Reply:
I just tried to use moment-timezone to set the Europe/Paris timezone, but it doesn't work well with webpack: https://github.com/moment/moment-timezone/issues/449

I tried the solution of this thread but none of them worked. I'm a bit stuck